ext/phar/tests/SplFileInfo_openFile_write.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
```

Comment:
the 2 tests contain identical contents.. did you miss to alter one of them?

Reply:
Oops. What happened is that I was first working on this test, and then noticed the wrong return value.
So then I started writing a test in this file and then did save-as. Which is how we ended up with a duplicate.